test/basic/pom.xml

```diff
@@ -17,7 +17,6 @@
 
   <properties>
     <!-- This configuration allows WildFly to be ran with various extra subsystems, including an OpenAPI subsystem -->
```

Comment:
Please consider removing this comment too, thanks.

Reply:
Thank you, I have removed it now :)